topology/nhlt/intel/ssp-nhlt.c

```diff
@@ -0,0 +1,600 @@
+/*
+  Copyright(c) 2021 Intel Corporation
+  All rights reserved.
+
+  This program is free software; you can redistribute it and/or modify
+  it under the terms of version 2 of the GNU General Public License as
```

Comment:
"Ssp code is lifted from Sound Open Firmware (sof) code base, thus it
will have BSD-3 license."

???

Reply:
"ssp _blob generation_ code is lifted" then maybe... so the intel specific blob thing is from sof, generic acpi nhlt is just new from me.